core/tests/puppeteer-acceptance-tests/utilities/user/topic-manager.ts

```diff
@@ -512,11 +629,121 @@ export class TopicManager extends BaseUser {
     await this.expectInputValueToBe(topicMetaTagInput, metaTags);
   }
 
+  /**
+   * Add any number of questions to a particular skill.
+   */
+  async createQuestionsForSkill(
+    skillName: string,
+    questionCount: number
+  ): Promise<void> {
+    for (let i = 0; i < questionCount; i++) {
+      await this.addBasicAlgebraQuestionToSkill(skillName);
+    }
+  }
+
+  /**
+   * Click on the create new question button in the skill editor.
+   */
+  async clickOnCreateNewQuestionButtonInSkillEditor(): Promise<void> {
+    await this.clickOn(createQuestionButton);
+    await this.expectElementToBeVisible(questionEditorContainer);
+  }
+
+  /**
+   * Create a basic algebra question in the skill editor page.
+   */
+  async addBasicAlgebraQuestionToSkill(skillName: string): Promise<void> {
+    await this.openSkillEditor(skillName);
+    await this.clickOn(createQuestionButton);
+    await this.clickOn(textStateEditSelector);
+    await this.page.waitForSelector(richTextAreaField, {visible: true});
+    await this.type(richTextAreaField, 'Add 1+2');
+    await this.page.waitForSelector(`${saveContentButton}:not([disabled])`);
+    await this.clickOn(saveContentButton);
+
+    await this.clickOn(addInteractionButton);
+    await this.page.waitForSelector(interactionNumberInputButton, {
+      visible: true,
+    });
+    await this.page.evaluate(interactionNameDiv => {
+      const interactionDivs = Array.from(
+        document.querySelectorAll(interactionNameDiv)
+      );
+      const element = interactionDivs.find(
+        element => element.textContent?.trim() === 'Number Input'
+      ) as HTMLElement;
+      if (element) {
+        element.click();
+      } else {
+        throw new Error('Cannot find number input interaction option.');
+      }
+    }, interactionNameDiv);
+
+    await this.clickOn(saveInteractionButton);
+    await this.page.waitForSelector('oppia-add-answer-group-modal-component', {
```

Comment:
why not define this selector at the top of the file with the others? Also, can one of the form `.e2e-test-*` be used?

Reply:
Updated from 
```
this.page.waitForSelector('oppia-add-answer-group-modal-component', {
      visible: true,
    });
``` 
to `await this.expectModalTitleToBe('Add Response');`